### PR TITLE
feat: add user authentication cookies to JitsiMeetView

### PR DESCRIPTION
**Pull request by GleidsonDaniel**
## Proposed changes

add user authentication cookies to JitsiMeetView

## Issue(s)	

closes: #2286

## Types of changes

- [x] Bugfix (non-breaking change which fixes an issue)
- [ ] Improvement (non-breaking change which improves a current function)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Documentation update (if none of the other choices apply)

## Checklist

- [x] I have read the [CONTRIBUTING](https://github.com/RocketChat/Rocket.Chat/blob/develop/.github/CONTRIBUTING.md#contributing-to-rocketchat) doc
- [x] I have signed the [CLA](https://cla-assistant.io/RocketChat/Rocket.Chat.ReactNative)
- [x] Lint and unit tests pass locally with my changes
- [ ] I have added tests that prove my fix is effective or that my feature works (if applicable)
- [ ] I have added necessary documentation (if applicable)
- [ ] Any dependent changes have been merged and published in downstream modules

## Further comments

Jira: [SUP-453]

[SUP-453]: https://rocketchat.atlassian.net/browse/SUP-453?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: app/views/JitsiMeetView/index.tsx [hunk 1]
```diff
@@ -17,8 +20,31 @@ const JitsiMeetView = (): React.ReactElement => {
...
+	const serverUrl = useAppSelector(state => state.server.server);
 
 	const [authModal, setAuthModal] = useState(false);
+	const [cookiesSet, setCookiesSet] = useState(false);
+
+	const setCookies = async () => {
+		const date = new Date();
+		date.setDate(date.getDate() + 1);
+		const expires = date.toISOString();
+		const domain = serverUrl.split('http://')[1];
```
Comment (reinaldonetof) [inline app/views/JitsiMeetView/index.tsx:33]: are you sure always will be an http server?

File: app/views/JitsiMeetView/index.tsx [hunk 2]
```diff
@@ -71,25 +97,49 @@ const JitsiMeetView = (): React.ReactElement => {
...
 	);
 };
 
+const styles = StyleSheet.create({
+	container: {
+		flex: 1
+	},
+	webviewContainer: { flex: 1, backgroundColor: 'rgb(62,62,62)' },
+	loading: { alignItems: 'center', justifyContent: 'center' }
+});
```
Comment (reinaldonetof) [inline app/views/JitsiMeetView/index.tsx:143]: I understand that the prettier don't see the necessity to break the line, but can you break them?

File: app/views/JitsiMeetView/index.tsx [hunk 3]
```diff
@@ -17,8 +20,31 @@ const JitsiMeetView = (): React.ReactElement => {
...
 	const { goBack } = useNavigation();
+	const user = useAppSelector(state => getUserSelector(state));
+	const serverUrl = useAppSelector(state => state.server.server);
 
 	const [authModal, setAuthModal] = useState(false);
+	const [cookiesSet, setCookiesSet] = useState(false);
+
+	const setCookies = async () => {
+		const date = new Date();
+		date.setDate(date.getDate() + 1);
```
Comment (reinaldonetof) [inline app/views/JitsiMeetView/index.tsx:31]: Why only 1 ms? Will the expire time affect the cookie if the device is slow?

**Review by reinaldonetof — CHANGES_REQUESTED** [review]

File: app/views/JitsiMeetView/index.tsx (hunk 3 above)
Comment (GleidsonDaniel) [reply app/views/JitsiMeetView/index.tsx:31]: 1 DAY
[setDate](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date/setDate)

File: app/views/JitsiMeetView/index.tsx (hunk 1 above)
Comment (GleidsonDaniel) [inline app/views/JitsiMeetView/index.tsx:33]:
```suggestion
		const domain = serverUrl.replace(/^https?:\/\//, '');
```

**Review by reinaldonetof — APPROVED** [review]

**Pull request merged**